docs/kyma-environment-broker/04-01-cls-architecture.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Cloud Logging Service Architecture
+type: Architecture
+---
+
+Cloud Logging Service (CLS) is a managed logging service for shipping logs from Runtimes. It uses the Elastic Stack (open source logging platform Elasticsearch, Logstash, Kibana) to store, parse, and visualize the application log data coming from Kubernetes applications.
```

Comment:
It is an SAP specific service and it is using FluentD instead of Logstash and it can handle application logs not only from kubernetes based applications